src/scripts.js

```diff
@@ -205,7 +205,7 @@ function showSavedRecipes() {
     let domRecipe = document.getElementById(`${recipe.id}`);
     domRecipe.style.display = "none";
   });
-  // showMyRecipesBanner();
+  showMyRecipesBanner();
```

Comment:
Turns out we do need this. Let's try and add into other functions to get this show up more for the user.